src/Parser.php

```diff
@@ -3516,10 +3516,7 @@ private function parseInterfaceElementFn() {
                     return $this->parseAttributeStatement($parentNode);
 
                 default:
-                    $missingInterfaceMemberDeclaration = new MissingMemberDeclaration();
-                    $missingInterfaceMemberDeclaration->parent = $parentNode;
-                    $missingInterfaceMemberDeclaration->modifiers = $modifiers;
-                    return $missingInterfaceMemberDeclaration;
+                    return $this->parseRemainingPropertyDeclarationOrMissingMemberDeclaration($parentNode, $modifiers);
```

Comment:
possible to add a test for this? or what is a code example that would invoke this change?